dokku

```diff
@@ -91,7 +91,7 @@ case "$1" in
 
     # if we can't post-deploy successfully, kill new container
     kill_new() {
-      docker inspect $id &> /dev/null && docker kill $id > /dev/null
+      docker inspect $id &> /dev/null && docker stop $id > /dev/null && docker kill $id > /dev/null
```

Comment:
You don't need `docker kill` as `docker stop` already sends a SIGKILL after 10 seconds<sup>1</sup>.

1. https://docs.docker.com/reference/commandline/cli/#stop